worldengine/generation.py

```diff
@@ -200,7 +200,7 @@ def generate_world(w, step):
 
     # Prepare sufficient seeds for the different steps of the generation
     rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
-    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
+    sub_seeds = rng.randint(0, 2147483647, size=100)  # sys.maxsize didn't quite work
```

Comment:
I'm using 32-bit python because some other libraries I'm using (libtcod, I think, and perhaps pygame) don't support 64-bit python. 

I can't check right now, but I'm using whatever the latest windows binary numpy has on sourceforge. 

I could revert this change, and then keep it changed locally whenever I want to use this library.